content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -33,79 +33,88 @@ further_reading:
 
-**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
+<div class="alert alert-warning"> 
+  <strong>Note:</strong> The .NET runtime tries to load a profiler into any .NET process started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. Don't set these environment variables globally because this causes all .NET processes on the host to load the profiler.
+</div>
+
+Set these environment variables for the non-IIS applications you want to instrument:
 
 | Name                   | Value                                    |
 | ---------------------- | ---------------------------------------- |
 | `COR_ENABLE_PROFILING` | `1`                                      |
 | `COR_PROFILER`         | `{846F5F1C-F9AE-4B07-969E-05C26BC060D8}` |
 
-#### Windows Services
 
-To automatically instrument a Windows Service, set the environment variables for that Service in the Windows Registry. Create a multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\<SERVICE NAME>` key. Then set the key's data to the values in the table:
-```text
-COR_ENABLE_PROFILING=1
-COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
-```
+#### Windows services
+To automatically instrument a Windows Service, set the `COR_ENABLE_PROFILING` and `COR_PROFILER` environment variables:
+
+1. In the Windows Registry Editor, create a multi-string value named `Environment` in  `HKLM\System\CurrentControlSet\Services\<SERVICE NAME>`
+2. Set the value data to: 
 
-This can be done either through the Registry Editor as in the image below, or through a PowerShell snippet:
+   ```text
+   COR_ENABLE_PROFILING=1
+   COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+   ```
+     {{< img src="tracing/setup/dotnet/RegistryEditorFramework.png" alt="Registry Editor"  >}}
 
-{{< img src="tracing/setup/dotnet/RegistryEditorFramework.png" alt="Registry Editor"  >}}
+Alternatively, you can set the environment variables by using the following PowerShell snippet:
 
-{{< code-block lang="powershell" filename="add-env-var.ps1" >}}
-[String[]] $v = @("COR_ENABLE_PROFILING=1", "COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}")
-Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<NAME> -Name Environment -Value $v
-{{< /code-block >}}
+   {{< code-block lang="powershell" filename="add-env-var.ps1" >}}
+   [String[]] $v = @("COR_ENABLE_PROFILING=1", "COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}")
+   Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<NAME> -Name Environment -Value $v
+   {{< /code-block >}}
 
-#### Console Apps
+#### Console applications
```

Comment:
Earlier there was a "non-IIS applications" section that seemed like it would have this information. I think the two sections should be combined or, if possible, make "Windows services" and "Console applications" sub-headings underneath "non-IIS applications"